triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/TriggerRunner.scala

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.lf.engine.trigger
+
+import akka.actor.typed.{Behavior, PostStop}
+import akka.actor.typed.scaladsl.AbstractBehavior
+import akka.actor.typed.SupervisorStrategy._
+import akka.actor.typed.Signal
+import akka.actor.typed.PostStop
+import akka.actor.typed.scaladsl.Behaviors
+import akka.actor.typed.scaladsl.ActorContext
+import akka.stream.Materializer
+import com.typesafe.scalalogging.StrictLogging
+import com.daml.grpc.adapter.ExecutionSequencerFactory
+
+object TriggerRunner {
```

Comment:
This is a super confusing name. We already have a class called `Runner`.

Reply:
The relationships are :  `TriggerRunner` has-a `TriggerRunnerImpl` has-a `Runner`.